### PR TITLE
Add validation and CI integration

### PR DESCRIPTION
**Pull request by mohsen1**

### CODE REVIEW

**Comment by mohsen1** [comment]
@adriancole TravisCI will not use this branch's `.travis.yaml` for testing it. After you merge, this will pass

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openzipkin/zipkin-api.git"
+  },
+  "keywords": [
+    "zipkin",
+    "api"
+  ],
+  "author": "",
+  "license": "ISC",
```
Comment (adriancole) [inline package.json]: Why is this ISC license? All of zipkin things are ASL. I'm worried we will have the burden of explaining this.

Comment (mohsen1) [inline package.json]: npm default!

Comment (adriancole) [inline package.json]: ok, if we aren't publishing to an npm repo, let's switch this to ASL, or just drop the key, if you don't mind. I'd like the license surface area of zipkin to be as small as possible. In other words, let's choose our battles.

**Comment by mohsen1** [comment]
Fixed the license

**Comment by adriancole** [comment]
great Thanks again!

**Pull request merged**